TODO.md

```
@@ -14,4 +14,6 @@
 - [ ] Use [pqoqubbw animated icons](https://icons.pqoqubbw.dev/).
 - [ ] Add a smooth dark/light mode view transition like [0.email](https://0.email).
 - So, when a user is re-putting their email it might mean they like to resubscribe, so update to unsubscribe: false
-- look at https://www.joshwcomeau.com/react/wonderful-emails-with-mjml-and-mdx/
\ No newline at end of file
+- look at https://www.joshwcomeau.com/react/wonderful-emails-with-mjml-and-mdx/
+- Marquee footer like https://21st.dev/serafimcloud/marquee/default
+- fix flex-1 not as it is grid
```

markdownlint-cli2 (0.17.2)

Files should end with a single newline character
null

(MD047, single-trailing-newline)